### PR TITLE
hooliPopup first commit

### PR DESCRIPTION
**Pull request by YuraZagor**
### task name

[Demo](https://yurazagor.github.io/) |
[Code base](https://github.com/YuraZagor/YuraZagor.github.io)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Review by expero1 — COMMENTED** [review]
Hi. You must add only index.html and style.css files in pull request for this task.

File: submissions/YuraZagor/hooliPopup/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+				<section class="outer-popup-container">
+					<div class="inner-container">
+						<ul class="popups--container">
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/blogs.png" alt="articles" class="popup--item"> </a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/apps.png" alt="4-color circle" class="popup--item"></a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/calendar.png" alt="calc" class="popup--item"></a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/chat.png" alt="chat" class="popup--item"></a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/classroom.png" alt="something" class="popup--item"></a></li>
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/collection.png" alt="asterisk" class="popup--item"></a></li>
```
Comment (expero1) [inline submissions/YuraZagor/hooliPopup/index.html:27]: As I see in demo the proportions of icons in popup menu  is differs from original. There are [few methods to solve this problem](https://stackoverflow.com/questions/12991351/css-force-image-resize-and-keep-aspect-ratio).

Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html:27]: man, i really apreciate you trying to help me, yet it reads to me like some hints about Hadron Collider)))  I mean I read and tried those  "[few methods to solve this problem](https://stackoverflow.com/questions/12991351/css-force-image-resize-and-keep-aspect-ratio)."  I just did my best with this task spending so much time (about 10 days) to so little good, so I desided to do the rest of tasks on JS  and then return here later on, if mentors don't tell me to focus on CSS

**Comment by A-Ostrovnyy** [comment]
Is this PR ready for review? If yes please change the status from Draft to Ready for review.

File: submissions/YuraZagor/hooliPopup/index.html (hunk 1 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html:27]: seems now they look better, thanks

File: submissions/YuraZagor/hooliPopup/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+</head>
+<body>
+	<nav>
+		<ul class="ul--menu">
+			<li class="li--menu"><a href="#">Mail</a></li>
+			<li class="li--menu"><a href="#">Images</a></li>
+
+			<li class="li--menu popup pic" >
+				<label id="popup-label" class="sub-menu" for='checkbox-icon'> <img src="./popup-icons/popup-button.png" alt="popup-label" > </label>
+				<input type="checkbox" id="checkbox-icon" name='checkbox-icon' class='checkbox-icon' alt="popup-checkbox" >
```
Comment (A-Ostrovnyy) [inline submissions/YuraZagor/hooliPopup/index.html:17]: Add styles on the focus state, to notify the user where the page focus is. [Watch video](https://share.vidyard.com/watch/cJKtB6pUbCL6JqXeP6gZif?)

File: submissions/YuraZagor/hooliPopup/index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+				<section class="outer-popup-container">
+					<div class="inner-container">
+						<ul class="popups--container">
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/blogs.png" alt="articles" class="popup--item"> </a></li>
```
Comment (A-Ostrovnyy) [inline submissions/YuraZagor/hooliPopup/index.html:22]: Please fix styles on focused state [Watch video](https://share.vidyard.com/watch/cJKtB6pUbCL6JqXeP6gZif?)

File: submissions/YuraZagor/hooliPopup/index.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+			
+			<li class="li--menu sub-menu" id="bell-label"><a href="#"><img src="./popup-icons/bell.png"  alt="bell"></a></li>
+			<li class="li--menu"><a href="#">login</a></li>
+
+		</ul>
+
+	</nav>
+
+
+	<link rel="stylesheet" href="./style.css">
```
Comment (A-Ostrovnyy) [inline submissions/YuraZagor/hooliPopup/index.html:59]: Move this link inside tag `head`

File: submissions/YuraZagor/hooliPopup/index.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+							<li class="li--popup--item"><a href="#"><img src="./popup-icons/photos.png" alt="two silouettes in ukranian colors" class="popup--item"></a></li>
+							<li class="li--popup--item"><a href="#"><img src="./popup-icons/promo.png" alt="something I can't get my head round" class="popup--item"></a></li>
+							<li class="li--popup--item"><a href="#"><img src="./popup-icons/tanslation.png" alt="useless stuff" class="popup--item"></a></li>
+							<li class="li--popup--item"><a href="#"><img src="./popup-icons/videos.png" alt="finally video" class="popup--item"></a></li>											
+						</ul>
+					</div>	
+				</section>
+			</li>
+			
+			<li class="li--menu sub-menu" id="bell-label"><a href="#"><img src="./popup-icons/bell.png"  alt="bell"></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/YuraZagor/hooliPopup/index.html:51]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/YuraZagor/hooliPopup/index.html (hunk 1 above)
Comment (A-Ostrovnyy) [reply submissions/YuraZagor/hooliPopup/index.html:27]: About aspect ratio better read [this article](https://jakearchibald.com/2022/img-aspect-ratio/)

File: submissions/YuraZagor/hooliPopup/style.css [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+
+.li--menu img {
+	width: 2.1em;
+	height: 2.1em;
+	cursor: pointer;
+}
+
+.sub-menu img {
+	width: 1.1em;
+	/* height: 1.1em; */
```
Comment (A-Ostrovnyy) [inline submissions/YuraZagor/hooliPopup/style.css:33]: Avoid commented code in your work.

File: submissions/YuraZagor/hooliPopup/style.css [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+	justify-content: flex-end;
+	align-items: center;
+	
+}
+.li--menu {
+	padding: 10px;
+	
+}
+
+.li--menu img {
```
Comment (A-Ostrovnyy) [inline submissions/YuraZagor/hooliPopup/style.css:25]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Avoid too specific CSS selectors. [Read about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Comment (A-Ostrovnyy) [inline submissions/YuraZagor/hooliPopup/style.css:25]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Avoid too specific CSS selectors. [Read about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/YuraZagor/hooliPopup/index.html (hunk 1 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html:28]: the trick with 	'object-fit: cover;'  helped make video-icon look good, thanks

Comment (A-Ostrovnyy) [reply submissions/YuraZagor/hooliPopup/index.html:28]: Better use `object-fit: contain` to display full icon)

File: submissions/YuraZagor/hooliPopup/index.html (hunk 4 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html]: done

File: submissions/YuraZagor/hooliPopup/style.css (hunk 7 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/style.css]: changed to selection by class

File: submissions/YuraZagor/hooliPopup/index.html (hunk 1 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html:28]: yep, even better, thanks a lot

File: submissions/YuraZagor/hooliPopup/index.html (hunk 2 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html]: I really tried a bunch of aproaches to outline the menu.  Finally I subjected both labels to  the  .show--all--label  ruleset  -- as it works for _.show--all_ input  -- and it wouldn't work for popup.  I tried by tag, by class and by id,   +  !important;  and it digitdn't help neither.  My logic breaks here how can one label work while the other doesn't

File: submissions/YuraZagor/hooliPopup/index.html (hunk 5 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html]: decided to remove alt texts for icons

File: submissions/YuraZagor/hooliPopup/style.css [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+	appearance: none;		
+}
+
+/* .checkbox-icon {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	overflow:hidden;
+	clip: rect(0 0 0 0);
+} */
```
Comment (A-Ostrovnyy) [inline submissions/YuraZagor/hooliPopup/style.css:80]: Avoid commented code in your work

File: submissions/YuraZagor/hooliPopup/index.html (hunk 3 above)
Comment (A-Ostrovnyy) [reply submissions/YuraZagor/hooliPopup/index.html]: Still not fixed :cry: 
image

File: submissions/YuraZagor/hooliPopup/index.html (hunk 2 above)
Comment (A-Ostrovnyy) [reply submissions/YuraZagor/hooliPopup/index.html]: Change the order, the label should be after the input, and then using CSS you can add styles for the label.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

File: submissions/YuraZagor/hooliPopup/index.html (hunk 3 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html]: Sorry, was completely stuck on menu.   Seems now it's working okay.  As I got it empty hrefs  still had their reserved space  and their container needed tuning.

File: submissions/YuraZagor/hooliPopup/style.css (hunk 8 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/style.css]: cleaned the working mess, sorry abt that

File: submissions/YuraZagor/hooliPopup/index.html (hunk 2 above)
Comment (YuraZagor) [reply submissions/YuraZagor/hooliPopup/index.html]: Thanks, now it's working.  I plan to read more on that matter

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**